src/app.js

```diff
@@ -1,21 +1,48 @@
+/* eslint-disable no-console */
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
-
-module.exports = sum;
+const fs = require('fs');
+const path = require('path');
+
+const moveFile = (src, dest) => {
+  if (src === dest) {
+    console.log('Source and destination paths are the same');
+
+    return;
+  }
+
+  if (!fs.existsSync(src)) {
+    console.log('Source file does not exist');
+
+    return;
+  }
+
+  if (!fs.existsSync(path.dirname(dest))) {
+    console.log('Destination directory does not exist');
+
+    return;
+  }
+
+  const destIsDirectory = dest.endsWith('/');
+
+  if (destIsDirectory) {
+    const filename = path.basename(src);
+    const newFilePath = path.join(dest, filename);
+
+    try {
+      fs.renameSync(src, newFilePath);
+      console.log('File moved successfully');
+    } catch (err) {
+      console.error('Error moving file:', err);
+    }
+  } else {
+    try {
+      fs.renameSync(src, dest);
+      console.log('File renamed/replaced successfully');
+    } catch (err) {
+      console.error('Error renaming/replacing file:', err);
+    }
+  }
```

Comment:
you can use `early exit` (`return`) to not use `else`
```suggestion
    }
    
    return;
  }
  
  try {
    fs.renameSync(src, dest);
    console.log('File renamed/replaced successfully');
  } catch (err) {
    console.error('Error renaming/replacing file:', err);
  }
```